src/apps/frontend/services/account.service.ts

```diff
@@ -6,7 +6,7 @@ import APIService from './api.service';
 export default class AccountService extends APIService {
   getAccountDetails = async (): Promise<ApiResponse<Account>> => {
     const userAccessToken = new AccessToken(
-      JSON.parse(localStorage.getItem('access-token')) as JsonObject,
+      JSON.parse(localStorage.getItem('access-token') || 'null') as JsonObject,
```

Comment:
This is not right.

Why are we using `null` string as a backup? When caller is calling `getAccountDetails`, it is going to assume access token is present in local storage